### PR TITLE
wizard/oscap: kernel and services customizations

### PR DESCRIPTION
**Pull request by lavocatt**
This PR ports the work done on 40b1d4de to the V1 wizard.

### CODE REVIEW

File: src/Components/CreateImageWizard/formComponents/OscapProfileInformation.tsx [hunk 1]
```diff
@@ -29,46 +32,97 @@ const OscapProfileInformation = (): JSX.Element => {
...
+              <TextListItem
+                component={TextListItemVariants.dt}
+                className="pf-u-min-width"
+              >
+                Operating system:
+              </TextListItem>
+              <TextListItem component={TextListItemVariants.dd}>
+                {RELEASES.get(getState()?.values?.['release'])}
+              </TextListItem>
+              component={TextListVariants.dl}
```
Comment (lucasgarfield) [inline src/Components/CreateImageWizard/formComponents/OscapProfileInformation.tsx:67]: Delete this line please, bug from the V2 wizard version that got copied over (will fix it there in a follow-up PR).

File: src/Components/CreateImageWizard/formComponents/OscapProfileInformation.tsx [hunk 2]
```diff
@@ -29,46 +32,97 @@ const OscapProfileInformation = (): JSX.Element => {
...
+              <TextListItem
+                component={TextListItemVariants.dt}
+                className="pf-u-min-width"
+              >
+                Profile description:
+              </TextListItem>
+              <TextListItem component={TextListItemVariants.dd}>
+                {oscapProfileInfo.openscap?.profile_description}
+              </TextListItem>
+              component={TextListVariants.dl}
```
Comment (lucasgarfield) [inline src/Components/CreateImageWizard/formComponents/OscapProfileInformation.tsx:57]: Delete this line please, bug from the V2 wizard version that got copied over (will fix it there in a follow-up PR).

**Comment by lavocatt** [comment]
@jrusz can you have a look at why the tests are failing please? We will have to merge this PR this morning in any case, a demo depends on it this afternoon.

**Review by lucasgarfield — APPROVED** [review]
Thank you @lavocatt !

**Pull request merged**